server/src/test/java/org/opensearch/index/query/IntervalQueryBuilderTests.java

```diff
@@ -861,7 +861,7 @@ public void testRegexp() throws IOException {
         String json = "{ \"intervals\" : { \"" + TEXT_FIELD_NAME + "\": { " + "\"regexp\" : { \"pattern\" : \"te.m\" } } } }";
 
         IntervalQueryBuilder builder = (IntervalQueryBuilder) parseQuery(json);
-        Query expected = new IntervalQuery(TEXT_FIELD_NAME, buildRegexpSource("te.m", DEFAULT_FLAGS, null));
+        Query expected = new IntervalQuery(TEXT_FIELD_NAME, buildRegexpSource("te.m", DEFAULT_FLAGS, 0, null));
```

Comment:
By overloading `buildRegexpSource` as described above we should be able to keep this original call. At first look it's not clear what `0` does here and maintainers will have to constantly look back.